common/config/azure-pipelines/integration-validation.yaml

```diff
@@ -0,0 +1,66 @@
+# iTwin.js Core Integration Validation Build
+#
+# This integration test job currently runs on all supported operating systems and node versions of iTwin.js.
+# Runs against master on a weekly schedule, in the future this should be a required check before bumping versions.
+
+trigger: none
+pr: none
+
+schedules:
+- cron: "0 0 * * Sun"
+  displayName: Weekly Sunday build
+  branches:
+    include:
+    - master
+
+variables:
+  - group: iTwin.js non-secret config variables
+  - group: iTwin.js Integration Test Users
+
+jobs:
+  - job: Integration_Tests_Full
+    strategy:
+      maxParallel: 3
+      matrix:
+        Linux_node_12_7:
+          imageName: ubuntu-latest
+          nodeVersion: 12.17.0
+        Linux_node_12_x:
+          imageName: ubuntu-latest
+          nodeVersion: 12.x
+        Linux_node_14_x:
+          imageName: ubuntu-latest
+          nodeVersion: 12.x
```

Comment:
Didn't catch this on the other PRs but this is set to 12 not 14

Reply:
Fixed this here and in the other backport, going to open a new PR into master to fix this shortly as well.